build.gradle

```diff
@@ -125,7 +125,7 @@ subprojects {
   }
 
   test {
-    maxHeapSize = "1024m"
+    maxHeapSize = "2048m"
```

Comment:
I was able to decrease the maxHeapSize to 1408m from 2048m.  This was due to the fix with the passthrough server.